lib/dradis/plugins/veracode/importer.rb

```diff
@@ -25,15 +30,20 @@ def import(params = {})
       end
 
       # create app_name, and parse attributes
-      node = parse_report_details(xml.root)
+      @node = parse_report_details(xml.root)
 
       # parse each severity > category > cwe > flaws
       xml.root.xpath('./xmlns:severity').each do |xml_severity|
         logger.info { "\t => Severity (level: #{ xml_severity[:level] })" }
         xml_severity.xpath('.//xmlns:flaw').each do |xml_flaw|
-          parse_flaw(xml_flaw, node)
+          parse_flaw(xml_flaw)
         end
       end
+
+      # parse each software_composition_analysis > ... > vulnerability
+      xml.root.xpath('.//xmlns:vulnerability').each do |xml_vuln|
```

Comment:
Do we really need to go back all the way to `xml.root`? And if so, us `//`? That should be reserved for when the search item is in multiple different parts of the XML structure because it's recursive and time consuming.

If the vulnerability items are in a predictable location (sca>etc.>) we can do better than this catch-all.

(and at the same time fix L38)

Reply:
Updated this to search based on the specific nesting we expect 
```
xml.root.xpath( 'xmlns:software_composition_analysis/xmlns:vulnerable_components//xmlns:vulnerability')
```

and updated L38 to do the same

```
xml_severity.xpath('./xmlns:category/xmlns:cwe/xmlns:staticflaws/xmlns:flaw')
```